### PR TITLE
CO : Correct phpDoc

### PR DESCRIPTION
**Pull request by djbuch**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | Simply correct phpDoc and ensure the return value is an int.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18182)

### CODE REVIEW

File: classes/order/Order.php [hunk 1]
```diff
@@ -1131,7 +1131,7 @@ public static function getCustomerNbOrders($id_customer)
                     . Shop::addSqlRestriction();
         $result = Db::getInstance(_PS_USE_SQL_SLAVE_)->getRow($sql);
 
-        return isset($result['nb']) ? $result['nb'] : 0;
+        return isset($result['nb']) ? (int)$result['nb'] : 0;
```
Comment (Progi1984) [inline classes/order/Order.php]:
```suggestion
        return isset($result['nb']) ? (int) $result['nb'] : 0;
```

Comment (Progi1984) [reply classes/order/Order.php]: Small fix. But :ok_hand: for me.

**Review by PierreRambaud — DISMISSED** [review]
PHP Cs fixer must be ran

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by Progi1984** [comment]
Only PHPDoc. No need QA.

**Comment by matks** [comment]
> Only PHPDoc. No need QA.

Plus a type casting 😄this could break some bad code.
However I checked for this function usage in standard prestashop install and found only usages into "Search (customers) page" and "GDPR Customer Search page".

**Comment by Progi1984** [comment]
Fail of my part :(

**Pull request merged**